nss-day-cohort-48/rare-client-orchesta-outlaws
# (VC) Homepage

- Added `getSubbedUserPosts` to `PostProvider.js`
- Factored Madeline's MyPosts render out into `PostList.js`
- Tweaked `Post.css` to make posts display vertically with a bottom margin
- Added `SubbedPosts` component and routed it to "/" in `ApplicationViews.js`

TO TEST:

- Checkout to main on the backend
- Run `rare-init` and `steve-subs` sql files to seed the database
- Log in as Steve. You should see two of the three new user's posts according to the subscriptions set up in `steve-subs.sql`
